libfuse/dir.go

```diff
@@ -347,6 +353,12 @@ func (d *Dir) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.Lo
 	d.folder.fs.log.CDebugf(ctx, "Dir Lookup %s", req.Name)
 	defer func() { d.folder.reportErr(ctx, libkbfs.ReadMode, err) }()
 
+	err = libkbfs.EnterCriticalWithTimeout(
```

Comment:
It has a timeout, which is why I didn't put an `ExitCritical` explicitly. But now thinking about it more, do we have code that relies on `<-ctx.Done()` to cleanup, and requres us to make sure `<-ctx.Done()` happens in time?